ci/test_wheel.sh

```diff
@@ -22,7 +22,7 @@ RAPIDS_TESTS_DIR=${RAPIDS_TESTS_DIR:-"${PWD}/test-results"}/
 mkdir -p "${RAPIDS_TESTS_DIR}"
 
 rapids-logger "Install wheel"
-python -m pip install --find-links ./dist pynvjitlink${PACKAGE_CUDA_SUFFIX}
+python -m pip install $(echo ./dist/pynvjitlink*.whl)
```

Comment:
Do we still want the CUDA suffix? Or does that matter?

If we don't need the suffix, should we drop `PACKAGE_CUDA_SUFFIX` from above?

```suggestion
python -m pip install $(echo ./dist/pynvjitlink-${PACKAGE_CUDA_SUFFIX}*.whl)
```

Reply:
I pushed a change for this.